modules/ROOT/pages/exchange-api.adoc

```diff
@@ -274,6 +275,32 @@ curl -v \
 
 To use the Exchange API version 1 to create an asset and set its lifecycle state, send the status property as a field.
 
+[[upload-asset-icons]]
+=== Upload Asset Icons
```

Comment:
```suggestion
=== Uploading Images to be Used as Asset Icons
```
Suggesting this to avoid the imperative voice in the title, which typically indicates that the content is a task, with steps. Also to be more precise: you upload the images, which are then used as icons.

Reply:
Do we need to say anything about why the audience wants or needs to do this: advantage?

Reply:
i can add that these icons display in Exchange which may be implied though.